test/common/index.js

```diff
@@ -551,25 +551,13 @@ function canCreateSymLink() {
   return true;
 }
 
-function getCallSite(top) {
-  const originalStackFormatter = Error.prepareStackTrace;
-  Error.prepareStackTrace = (err, stack) =>
-    `${stack[0].getFileName()}:${stack[0].getLineNumber()}`;
-  const err = new Error();
-  Error.captureStackTrace(err, top);
-  // With the V8 Error API, the stack is not formatted until it is accessed
-  err.stack; // eslint-disable-line no-unused-expressions
-  Error.prepareStackTrace = originalStackFormatter;
-  return err.stack;
-}
-
 function mustNotCall(msg) {
-  const callSite = getCallSite(mustNotCall);
+  const callSite = getCallSite()[1];
   return function mustNotCall(...args) {
     const argsInfo = args.length > 0 ?
       `\ncalled with arguments: ${args.map((arg) => inspect(arg)).join(', ')}` : '';
     assert.fail(
-      `${msg || 'function should not have been called'} at ${callSite}` +
+      `${msg || 'function should not have been called'} at ${callSite.scriptName}:${callSite.lineNumber}` +
```

Comment:
This only prints a single frame without the function name. The original gave more info, and included more frames.

Reply:
Got it, I'll update the file today

Reply:
I'm not sure that's the case, AFAICT the old `getCallSite` only got one frame (`stack[0]`).

Reply:
You are right

https://github.com/nodejs/node/blob/f8b7a171463e775da304bccf4cf165e634525c7e/test/common/index.js#L557